pages/index.js

```diff
@@ -20,6 +21,11 @@ export default function Home({
   const [activeChartData, setActiveChartData] = useState(true);
   const [selectedChart, setSelectedChart] = useState(null);
   const [toggleForm, setToggleForm] = useState(false);
+  const [filter, setFilter] = useState("");
```

Comment:
The naming of this state to save the search term is a little bit confusing. Maybe searchTerm or searchQuery is a better name?

Reply:
and also done 😄